src/plugins/vis_builder/public/application/utils/get_top_nav_config.tsx

```diff
@@ -183,6 +183,10 @@ export const getOnSave = (
     savedVisBuilderVis.title = newTitle;
     savedVisBuilderVis.description = newDescription;
     savedVisBuilderVis.copyOnSave = newCopyOnSave;
+    const searchSourceInstance = savedVisBuilderVis.searchSourceFields;
```

Comment:
Is `searchSourceFields` always a simple object?

Reply:
Yes